SpotifyArtistsViewer/SAArtistViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SAArtistViewController.swift
+//  SpotifyArtistsViewer
+//
+//  Created by Kevin Zeckser on 6/6/16.
+//  Copyright © 2016 Kevin Zeckser. All rights reserved.
+//
+
+import UIKit
+
+class SAArtistViewController: UIViewController {
+    
+    // MARK: - Properties & Outlets
+    
+    @IBOutlet weak private var artistImageView: UIImageView!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    var url :String?
+    var detailArtist: SAArtist?
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.navigationBar.barTintColor = UIColor.blackColor()
```

Comment:
Also, you can remove the 'UIColor' here, as the type is inferred from the left side. Keeps it a little cleaner.